source/network/palos-troubleshooting.html.md.erb

```diff
@@ -149,4 +149,44 @@ When this happens then there needs to be a cleaning exercise or a hard reboot as
 Link with further details
 
 - [High Disk Space Usage on / root partition and How To Clear](https://knowledgebase.paloaltonetworks.com/KCSArticleDetail?id=kA10g000000ClaJCAS)
-- [AnalysisMan](https://www.analysisman.com/2020/08/pan-diskspace.html)
\ No newline at end of file
+- [AnalysisMan](https://www.analysisman.com/2020/08/pan-diskspace.html)
+
+## Support Tickets with Palo Alto
+
+### **Pre-req**
+You need to have a Palo Alto Customer Support Portal (SCP) account already setup, if not then you can create one in the [CSP](https://support.paloaltonetworks.com/Support/Index/).
+An already existing user will need to assign you to the account, reach out to the team on the #platform-operations Slack channel to get this done.
+
+You will also need Google Authenticator or similar 2FA app to login to the CSP.
+
+### **Creating a new ticket**
+Follow the steps `Support` -> `Get Help` then select what type of ticket you want to create and provide the necessary information.
+
+You can also add attachments to the ticket after creating it, so if you have a tech support file you can attach it here or any other relevant information requested.
+
+<details>
+    <summary>Step one</summary>
+    <img alt="Step one" src="images/tac-step-1.png">
+</details>
+
+<details>
+    <summary>Step two</summary>
+    <img alt="Step two" src="images/tac-step-2.png">
+</details>
+
+<details>
+    <summary>Step three</summary>
+    <img alt="Step three" src="images/tac-step-3.png">
+</details>
+
+### **Things to Note**
+- There is a good chance they will ask for the tech support file, so make sure you have that ready
+  See the generating a tech support file section in the [Palo Alto Software Upgrade](./palos-upgrade.html) [Prerequisite](./palos-upgrade.html#prerequisite) guide
+- You will need to factor timezone differences when arranging a call with them, make sure you are specific about your timezone
+- They may ask for details like serial number, software version, model etc. Make sure you have that information to hand
```

Comment:
how/where?

Reply:
same link just above @thomast1906 on line 184 but adding for more verbosity